marklogic-data-hub-central/ui/src/components/modeling/property-modal/property-modal.tsx

```diff
@@ -858,7 +858,7 @@ const PropertyModal: React.FC<Props> = (props) => {
           colon={false}
           labelAlign="left"
           validateStatus={errorMessage ? "error" : ""}
-          help={errorMessage}
+          help={errorMessage === "name-error" ? <span data-testid="propery-name-error">A property or structured type are already using the name <b>{name}</b>. A property cannot use the same name as an existing property or structured type.</span> : errorMessage}
```

Comment:
Looks like the testid has a typo. Should be "property-name-error" instead of "propery-name-error". The RTL and Cypress tests referring to this test-id should be fixed as well.